test/fixture/locales/US/en/complex.properties

```diff
@@ -0,0 +1,5 @@
+value=A Value with {numPhotos, plural, =0 {no photos.} =1 {one photo.} other {# photos.}}
+array[0]=First Entry
+array[1]=Second Entry
+map[US]=United States
+map[GB]=United Kingdom
```

Comment:
For testing plurals, it would be good to see some [additional plural examples](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals), particularly for Chinese, French, and Russian.  If you can handle those cases, you should be able to handle any language.

Note that there are more plural cases than just zero, 1, plural.  The Yahoo! intl-messageformat uses [ICU plural syntax](http://icu-project.org/apiref/icu4j/com/ibm/icu/text/PluralRules.html) which allows modulo rules, and enables scenarios such as 1st, 2nd, 3rd, 4th, 5th, {n}th ... 20th, 21st, 22nd, 23rd, 24th, {n}th, ... etc. and equivalent in Russian, Arabic, Thai, etc.

Anyway, it would be good to see a few test cases with plural rules that differ from English and have a different number of selectable strings, depending on count.